theme/islandora-dublin-core-description.tpl.php

```diff
@@ -15,6 +15,6 @@
 <div class="islandora-metadata-sidebar">
   <?php if (!empty($dc_array['dc:description']['value'])): ?>
     <h2><?php print $dc_array['dc:description']['label']; ?></h2>
-    <p property="description"><?php print $dc_array['dc:description']['value']; ?></p>
+    <p property="description"><?php print nl2br(filter_xss($dc_array['dc:description']['value'])); ?></p>
```

Comment:
Good call. I didn't notice that `filter_xss` was being called in the DC class. I think there is an argument for having `filter_xss` to be happening very close to where it is displayed, so its obvious that the variable is filtered, but on the other hand, its not really great to have it called twice.

So would it make sense to remove the call to `filter_xss` and move the call to `nl2br` to `islandora_preprocess_islandora_dublin_core_description()`?